controllers/managedmcg_controller.go

```diff
@@ -565,6 +582,9 @@ func (r *ManagedMCGReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	err := ctrl.NewControllerManagedBy(mgr).
 		For(&mcgv1alpha1.ManagedMCG{}, managedMCGPredicates).
+		// Watch owned resource
```

Comment:
do we need these two comments, the code is self-explanatory?

Reply:
ACK, removing.